test/manipulation.test.js

```diff
@@ -756,54 +756,65 @@ describe('manipulation', function () {
     it('should destroy filtered set of records');
   });
 
-  describe('deleteAll/destroyAll', function () {
-    beforeEach(function clearOldData(done) {
-      Person.deleteAll(done);
+  describe('when deleteAll/destroyAll executes', function () {
+    it('should be defined as function', function() {
+      Person.deleteAll.should.be.a.Function;
+      Person.destroyAll.should.be.a.Function;
     });
 
-    beforeEach(function createTestData(done) {
-      Person.create([
-        { name: 'John' },
-        { name: 'Jane' }
-      ], done);
-    });
+    describe('with multiple instances in the database', function() {
+      beforeEach(function deleteFixtures(done) {
+        Person.deleteAll(done);
+      });
 
-    it('should only delete instances that satisfy the where condition', function (done) {
-      Person.deleteAll({name: 'John'}, function (err, result) {
-        if (err) return done(err);
-        result.should.have.property('count', 1);
-        Person.find({where: {name: 'John'}}, function (err, data) {
+      beforeEach(function createFixtures(done) {
+        Person.create([{
+          name: 'John'
+        }, {
+          name: 'Jane'
+        }], done);
```

Comment:
Please preserve the formatting next time, it makes patches easier to review.

Also AFAIK we don't use the style you introduced above, which is another reason for not reformatting code to this new style.